internal/core/openapi/openapi-ng/auth/over_permission/match/form_body_value.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package match
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+
+	"github.com/erda-project/erda/pkg/encoding/jsonparse"
+)
+
+func init() {
+	registry("form_body", formBodyValue{})
```

Comment:
maybe it's unnecessary to create a new provider called `org-id`, just register `form_body` in `openapi-over-permission-org` is enough and clearer.